.gitlab-ci.yml

```diff
@@ -2,7 +2,7 @@ variables:
   # These are gitlab variables so that it's easier to do a manual deploy
   # If these are set witih value and description, then it gives you UI elements
   DOWNSTREAM_BRANCH:
-    value: "main"
+    value: "ban/switch-to-nextest" # TODO switch back
```

Comment:
Just calling this out in case you meant switch it back before posting for review.